lib/components/icons/SpaceIcon.tsx

```diff
@@ -0,0 +1,115 @@
+import { FC } from 'react';
+
+const SpaceIcon: FC = () => (
+  <svg viewBox="0 0 215 163" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <g clip-path="url(#clip0_2283_14148)">
```

Comment:
think this might have to be changed to `clipPath` but I'm actually not sure—if it breaks smth then keep as-is.

```suggestion
    <g clipPath="url(#clip0_2283_14148)">
```